code/src/components/MessageElement.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const MessageElement = ({ item, onHeartsIncrease }) => {
+    return (
+        <>
+            {/* <div key={item._id}> */}
+                <h3>{item.message}</h3>
+                <p>Created at: {item.createdAt}</p>
+                <button onClick={() => onHeartsIncrease(item._id)}> 
```

Comment:
Your button should have a different background color if there are any likes. Please implement this logic.